api/models/post/createPost.js

```diff
@@ -33,30 +33,30 @@ export function afterCreatingPost (post, opts) {
 
     // Add media, if any
     // redux version
-    opts.imageUrl && Media.createForPost({
+    opts.imageUrl && Media.createForSubject({
       postId: post.id, type: 'image', url: opts.imageUrl
```

Comment:
could just use subjectId and subjectType here right?

Reply:
Yes, I introduced that pattern as an option and used it throughout in Comment, but left support for this legacy way of doing things intact for Post as it's relied upon at a couple of layers. I guess I could fully refactor to this pattern throughout.

Reply:
seems like it should be easy to do, if so do it, if its a big pain, no worries about it